jxl/src/image.rs

```diff
@@ -117,6 +117,13 @@ impl<T: ImageDataType> Debug for Image<T> {
         write!(f, "{:?} {}x{}", T::DATA_TYPE_ID, self.size.0, self.size.1,)
     }
 }
+
+#[derive(Debug)]
+pub struct Rect {
```

Comment:
If we introduce this struct, might it make sense to change ImageRect to have Rect as a field, instead of repeating origin and size?